static/src/js/components/Errors/NotFound.js

```diff
@@ -1,59 +1,57 @@
-import React, { Component } from 'react'
+import React, { memo, useEffect } from 'react'
 import { v4 as uuidv4 } from 'uuid'
 
 import { eventEmitter } from '../../events/events'
 import LoggerRequest from '../../util/request/loggerRequest'
 import { locationPropType } from '../../util/propTypes/location'
 
-class NotFound extends Component {
-  UNSAFE_componentWillMount() {
+export const NotFound = memo(({
```

Comment:
Do we need to use `memo` on these components? I thought that was for replacing PureComponent?

Reply:
oops, it is. I had read that memoizing is good for performance for functional components in general but I just looked into it again and it's more recommended for larger components or ones that render often

Reply:
I had wondered about this but thought you might have been given direction to add it. `memo` is probably overkill for these components.